test/unit/metrics/test_group_metric_set.py

```diff
@@ -85,27 +86,20 @@ def test_group_names_values_not_string(self):
 
     def test_metrics(self):
         target = GroupMetricSet()
-        my_metric = GroupMetricResult()
+        my_metric = Bunch()
         my_metric.overall = 10222
         target.metrics = {"60bdb14d-83fb-4374-ab2e-4c371f22b21e": my_metric}
         assert target.metrics["60bdb14d-83fb-4374-ab2e-4c371f22b21e"].overall == 10222
 
     def test_metrics_keys_not_string(self):
         target = GroupMetricSet()
-        my_metric = GroupMetricResult()
+        my_metric = Bunch()
         my_metric.overall = 10222
         with pytest.raises(ValueError) as exception_context:
             target.metrics = {0: my_metric}
         expected = "Keys for metrics dictionary must be strings"
         assert exception_context.value.args[0] == expected
 
-    def test_metrics_values_not_groupmetricresult(self):
-        target = GroupMetricSet()
-        with pytest.raises(ValueError) as exception_context:
-            target.metrics = {"a": 0}
-        expected = "Values for metrics dictionary must be of type GroupMetricResult"
-        assert exception_context.value.args[0] == expected
-
 
 class TestConsistencyCheck:
```

Comment:
curious, why are the tests in classes?

Reply:
Can't remember offhand why these ones are.

Reply:
they may be remnants of `nose` or some other testing tool. `pytest` doesn't need them.

Reply:
We've always been `pytest`, but haven't been consistent in whether we use test classes or not. It might be down to the fact that the `_metrics_engine` module had two routines, so I had a class for each of them. That would be a remnant from before we reorganised the namespaces.